tools/test.py

```diff
@@ -111,6 +111,10 @@ def main():
 
     # build the dataloader
     dataset = build_dataset(cfg.data.test)
+    limit = cfg.data.get('limit_eval')
+    if limit is not None:
+        dataset.reduce_to_subset(range(limit))
```

Comment:
What's the purpose of this function?

Reply:
The evalutaion is crashing here after running for an hour or so. This would allow me to debug it faster.